blocks/form/form.css

```diff
@@ -893,4 +906,35 @@ main .two-column.form-container .form-wrapper {
         padding: 13px 0;
         margin-top: 0;
     }
+
+    main .two-column-alternate.form-container {
+        display: table-row;
+    }
+
+}
+
+main .two-column-alternate {
```

Comment:
why alternate?

Reply:
instead of creating two different selectors, see if we can make it generic to have all n-1 div in 1 column and last div in 2nd column